### ISSUE TITLE
Turn Discord Webhook into a QtPlugin

### ISSUE THREAD

**Issue by Salanto** [Salanto/akashi#9]
Discoreds API is ever-evolving, making maintaining it a trial by itself as the implementation can become non-functional at any given moment without any warning. Turning it into a Plugin would give some flexibility on when people can update their server and allow quicker reactions for releases, as it would only be a library as opposed to the entire server (and potential new features that the server owner could not test yet) just to have working webhooks

TLDR: Turn Discord class into a Plugin to make it easier to update without doing a release.
